### PR TITLE
sunos/*: add Turkish translations

### PR DESCRIPTION
**Pull request by yutyo**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.tr/sunos/truss.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# truss
+
+> İzleme sistem çağrıları için sorun giderme aracı.
+> SunOS'in strace alternatifi.
+> Daha fazla bilgi için: <https://www.unix.com/man-page/linux/1/truss>.
+
+- Bir programı tüm alt işlemleriyle beraber çalıştırarak başlat:
+
+`truss -f {{program}}`
```
Comment (navarroaxel) [inline pages.tr/sunos/truss.md:9]: is `program` translatable?

**Review by navarroaxel — APPROVED** [review]

File: pages.tr/sunos/truss.md (hunk 1 above)
Comment (yutyo) [reply pages.tr/sunos/truss.md:9]: Program is used as program in Turkish language.

> is `program` translatable?

File: pages.tr/sunos/devfsadm.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# devfsadm
+
+> `/dev` için yönetim komutu. `/dev` ad alanına yönetir.
+> Daha fazla bilgi için: <https://www.unix.com/man-page/sunos/1m/devfsadm>.
```
Comment (kbdharun) [inline pages.tr/sunos/devfsadm.md:4]:
```suggestion
> Daha fazla bilgi: <https://www.unix.com/man-page/sunos/1m/devfsadm>.
```
Same doubt here.

Comment (yutyo) [reply pages.tr/sunos/devfsadm.md:4]: "Daha fazla bilgi için" is much more elegant. "Daha fazla bilgi" doesn't sound native but sounds like a direct English to Turkish translation done by a bot.

Comment (navarroaxel) [reply pages.tr/sunos/devfsadm.md:4]: But our standard translation is `Daha fazla bilgi` in https://github.com/tldr-pages/tldr/blob/main/scripts/set-more-info-link.py All the same, or change all Turkish pages

Comment (yutyo) [reply pages.tr/sunos/devfsadm.md:4]: All Turkish pages that lack "için" should have it. I will make a PR for that.

Comment (kbdharun) [reply pages.tr/sunos/devfsadm.md:4]: > All Turkish pages that lack "için" should have it. I will make a PR for that.

Great, I will add in my approval.

**Review by kbdharun — APPROVED** [review]

**Pull request merged**